internal/handlers/sqlite/msg_create.go

```diff
@@ -27,6 +28,9 @@ import (
 	"github.com/FerretDB/FerretDB/internal/wire"
 )
 
+// validateDatabaseNameRe validates FerretDB database name.
+var validateDatabaseNameRe = regexp.MustCompile("^[a-zA-Z_-][a-zA-Z0-9_-]{0,62}$")
+
```

Comment:
There's no need to additional validation for SQLite to satisfy validation that we have on other handlers